tests/Public/New-GuestConfigurationPolicy.Tests.ps1

```diff
@@ -690,6 +690,61 @@ Describe 'New-GuestConfigurationPolicy' {
                         Assert-GuestConfigurationPolicyDefinitionFileValid @assertionParameters
                     }
                 }
+
+                Context 'New parameters - contentManagedIdentity and contentPath' {
```

Comment:
I wouldn't call these new parameters.
Next year when we are working on this code, these aren't new parameters.
I would maybe just call them Identity parameters or mention that they are optional parameters.

Reply:
Changed the context to - Optional identity parameters - ManagedIdentityResourceId and contentPath